### PR TITLE
XRF auto fit

### PR DESCRIPTION
**Pull request by licode**
Add auto fit gui for fluorescence fitting.

### CODE REVIEW

File: bubblegum/xrf/view/guessparam.enaml [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+                param_m.find_peak()
+                param_m.arange_prefit_result()
+                status_tip = 'Running'
+                if (len(param_m.total_y) == 0):
+                    #lb15.text = 'No auto fit result is available.'
+                    pb13.enabled = False
+                    pb14.enabled = False
+
+                else:
+                    #lb15.text = 'Auto fit is finished.'
```
Comment (ericdill) [inline bubblegum/xrf/view/guessparam.enaml]: Do you want to remove all lines that are commented in here? There are quite a number

**Comment by licode** [comment]
Let's ignore this pr for now. I will send a new one with a better plotting part.

**Comment by ericdill** [comment]
... are you kidding me... I've been working on this PR for the last 5 hours

**Comment by licode** [comment]
Really sorry about that. I checked your PR, and find basically there is not so much conflicts with the code I updated. I mainly focus on the lineplot part. So nothing waste...Good work!

**Comment by licode** [comment]
I will send a new one with merged work.

**Pull request closed without merging**